doc/getting_started.rst

```diff
@@ -102,7 +102,7 @@ the test data::
   >>> # create a pipeline object
   >>> pipe = make_pipeline(
   ...     StandardScaler(),
-  ...     LogisticRegression(random_state=0)
+  ...     LogisticRegression()
```

Comment:
please update the printout below to fix doctests

Reply:
Ah! Sorry I missed that. Fixed!